index.html

```diff
@@ -349,6 +341,24 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>Returns the result of the
+           <a href="https://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">retargeting algorithm</a>
```

Comment:
Event dispatch's *retargeting* in DOM Standard did not need *retargeting* defined in Shadow DOM spec, *magically*.
Here, it is time to need a retargeting algorithm defined in Shadow DOM. This is the first time, I think.